lib/Console/Traits/Parallelization.php

```diff
@@ -33,6 +33,9 @@ trait Parallelization
 
     use ParallelizationBase;
 
+    /**
+     * @deprecated Deprecated since 2.0.0 and will be removed in 3.0.0. Use ParallelizationInput::configureCommand() instead.
```

Comment:
Version `2.0.0` and `3.0.0` of what?

Reply:
Added the bundle name to make it more clear.